code/game/objects/effects/spawners/structure.dm

```diff
@@ -16,10 +17,9 @@ again.
 //normal windows
 
 /obj/effect/spawner/structure/window
-	icon = 'icons/obj/structures_spawners.dmi'
 	icon_state = "window_spawner"
 	name = "window spawner"
-	spawn_list = list(/obj/structure/window_frame/reinforced/grille_and_window)
+	spawn_list = list(/obj/structure/window_frame/grille_and_window)
```

Comment:
@Jacquerel what's going on here?

Reply:
maybe im wrong but i think this was a typo
this is what it is on main tg:

```
/obj/effect/spawner/structure/window
	icon = 'icons/obj/structures_spawners.dmi'
	icon_state = "window_spawner"
	name = "window spawner"
	spawn_list = list(/obj/structure/grille, /obj/structure/window/fulltile)
	dir = SOUTH
```

the reinforced spawner has its own typepath below this one